src/entropy.jl

```diff
@@ -149,11 +149,11 @@ Here, ``S`` is the [Von Neumann entropy](https://en.wikipedia.org/wiki/Von_Neuma
 - `kwargs` are the keyword arguments for calculating Von Neumann entropy. See also [`entropy_vn`](@ref).
 """
 function entropy_mutual(
-    ρAB::QuantumObject{ObjType,<:AbstractDimensions{N}},
+    ρAB::QuantumObject{ObjType,<:AbstractDimensions{N,M}},
```

Comment:
Should we use `M` first and then `N` (alphabetical order) ? 
```suggestion
    ρAB::QuantumObject{ObjType,<:AbstractDimensions{M,N}},
```

Reply:
ok